src/main/cms/cms_menu_imu.c

```diff
@@ -517,6 +517,7 @@ static CMS_Menu cmsx_menuLaunchControl = {
 #endif
 
 static uint8_t  cmsx_angleStrength;
+static uint8_t  cmsx_angleFF;
```

Comment:
Rather than `angleFF`, `angleFeedforward` might be more consistent with avoiding FF elsewhere.

Reply:
Agreed. Changed.